app/frontend/actions.js

```diff
@@ -864,6 +937,70 @@ module.exports = ({setState, getState}) => {
     validateSendFormAndCalculateFee()
   }
 
+  const updateStakePoolAddress = (state, e) => {
+    setState({
+      stakePools: state.stakePools.map((pool) => {
+        return Object.assign(
+          {},
+          pool,
+          pool.ID === parseInt(e.target.id, 10)
```

Comment:
parseInt for pool.ID? they are hex values

Reply:
Removed.